### PR TITLE
Updated Twitter URL to x.com and added Bluesky profile link in footer (#2932)

### PR DESCRIPTION
**Pull request by Shubhashish-Chakraborty**
## Changes Made to the Footer Section:
- Updated the Twitter Profile URL from `twitter.com/babeljs` to `x.com/babeljs`.
- As per the request from @nicolo-ribaudo  ,  I have also added a link to the Bluesky account for BabelJS.

## Visual Overview:

- **Here's what the footer looks like after changes:**
PRresult

- **Here's what the udpated code looks like:**
babelPR

### CODE REVIEW

File: website/src/theme/Footer/index.js [hunk 1]
```diff
@@ -31,11 +31,18 @@ function Footer() {
...
           >
-            Twitter
+            X (Twitter)
+          </Link>
+          <Link
+            to="https://bsky.app/profile/babel.dev"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Bluesky Profile
```
Comment (nicolo-ribaudo) [inline website/src/theme/Footer/index.js:45]:
```suggestion
            Bluesky
```

**Review by nicolo-ribaudo — COMMENTED** [review]
Just a minor comment, but it looks good otherwise

**Review by existentialism — APPROVED** [review]

**Review by Shubhashish-Chakraborty — COMMENTED** [review]
**It is perfectly Fine Sir!**

**Comment by nicolo-ribaudo** [comment]
Thank you!

**Pull request merged**